tools/importer/importer.py

```diff
@@ -0,0 +1,252 @@
+import os, json, stat, sys, shutil, errno, subprocess, logging, re
+import argparse
+from os.path import dirname, abspath, basename, join
+
+# Be sure that the tools directory is in the search path
+ROOT = abspath(join(dirname(__file__), "../.."))
+sys.path.insert(0, ROOT)
+
+from tools.utils import run_cmd, delete_dir_files, mkdir, copy_file
+
+def del_file(name):
+    """ Delete the file in RTOS/CMSIS/features directory of mbed-os
+    Args:
+    name - name of the file
+    """
+    result = []
+    search_path = [join(ROOT, 'rtos'), join(ROOT, 'cmsis'), join(ROOT, 'features')]
+    for path in search_path:
+        for root, dirs, files in os.walk(path):
+            if name in files:
+                result.append(os.path.join(root, name))
+    for file in result:
+        os.remove(file)
+        rel_log.debug("Deleted: %s", os.path.relpath(file, ROOT));
+
+def copy_folder(src, dest):
+    """ Copy contents of folder in mbed-os listed path
+    Args:
+    src - src folder path
+    dest - destination folder path
+    """
+    files = os.listdir(src)
+    for file in files:
+        abs_src_file = os.path.join(src, file)
+        if os.path.isfile(abs_src_file):
+            abs_dst_file = os.path.join(dest, file)
+            mkdir(os.path.dirname(abs_dst_file))
+            copy_file(abs_src_file, abs_dst_file)
+
+def run_cmd_with_output(command, exit_on_failure=False):
```

Comment:
I think that this may be equivalent to `tools.utils.run_cmd_ext` with the return values moved around.

Reply:
run_cmd_ext - Checks if command exists or not. Here I need to terminate if command execution fails (not because it does not exists, but due to some other issue)

Reply:
Thanks for looking into that.